scipy/signal/spectral.py

```diff
@@ -398,6 +399,110 @@ def csd(x, y, fs=1.0, window='hanning', nperseg=256, noverlap=None, nfft=None,
     return freqs, Pxy
 
 
+def spectrogram(x, fs=1.0, window=('tukey',.25), nperseg=256, noverlap=None,
+                nfft=None, detrend='constant', return_onesided=True,
+                scaling='density', axis=-1):
+    """
+    Compute a spectrogram with consecutive Fourier transforms.
+
+    Parameters
+    ----------
+    x : array_like
+        Time series of measurement values
+    fs : float, optional
+        Sampling frequency of the `x` time series in units of Hz. Defaults
```

Comment:
I know `Hz` is used in a few other places in scipy.signal, but we should stop doing that.  There is nothing in the code that requires the units to be cycles per *second*.  Everything works just the same if the frequency is cycles per day, cycles per years, etc.